src/main/java/com/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,23 @@
 package com.onlinebookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
```

Comment:
It's better to use NotBlank for String

```suggestion
    @NotBlank
```